setup.py

```diff
@@ -83,7 +83,7 @@
         "console_scripts": ["smry2arrow_batch=webviz_subsurface.smry2arrow_batch:main"],
     },
     install_requires=[
-        "dash>=2.0.0",
+        "dash<2.4",
```

Comment:
should we also still force `>=2.0.0`

Reply:
Yes, @lindjoha gave same output, will change to `>=2.0, <2.4`.